test/policies/modifier/modifier.test.js

```diff
@@ -118,6 +120,14 @@ const setupGateway = () => {
                   remove: ['x-test']
                 }
               }
+            }, {
+              action: {
+                body: {
+                  add: {
+                    'foo.url': 'res.body.url'
```

Comment:
This is demonstrating that a new nested property can be set on the body, using some property of the original response body.  Even though there is a responseTransformer action above that removes the `url` property from the response body, we can access the original response body here and use the value of the original `url` property to set a new `foo.url` property in the transformed response body.